pkg/broker/awsbroker.go

```diff
@@ -357,9 +366,21 @@ func (db Db) ServiceDefinitionToOsb(sd CfnTemplate) osb.Service {
 }
 
 func (b *AwsBroker) generateS3HTTPUrl(serviceDefName string) *string {
-	prefix := "https://s3.amazonaws.com/"
-	if b.s3region != "us-east-1" {
-		prefix = fmt.Sprintf("https://s3-%s.amazonaws.com/", b.s3region)
+	if b.partition == "aws" {
+		// AWS Standard Partition
+		prefix := "https://s3.amazonaws.com/"
+		if b.s3region != "us-east-1" {
+			prefix = fmt.Sprintf("https://s3-%s.amazonaws.com/", b.s3region)
```

Comment:
It would be nice to update the url here to use virtual host style to match what you've done for the `aws-cn` partition

```go
prefix := fmt.Sprintf("https://%s.s3.amazonaws.com/", b.s3bucket)
    if b.s3region != "us-east-1" {
        prefix = fmt.Sprintf("https://%s.s3-%s.amazonaws.com/", b.s3bucket, b.s3region)
    }
    return aws.String(prefix + b.s3key + strings.TrimSuffix(serviceDefName, "-apb") + b.templatefilter)
```

Reply:
sure